projects/angular/src/accordion/stepper/stepper-panel.ts

```diff
@@ -45,6 +46,9 @@ export class ClrStepperPanel extends ClrAccordionPanel implements OnInit {
   private subscriptions: Subscription[] = [];
 
   constructor(
+    @Optional()
+    @SkipSelf()
+    public override parent: ClrAccordionPanel,
```

Comment:
Sam here. I think this should either be `protected` or `private`.

Reply:
It can be `protected`, but not `private`.

Reply:
Corrected in commit: cc43f46